notebooks/FAISS_with_🤗_datasets_and_CLIP.ipynb

```diff
@@ -0,0 +1,10919 @@
+{
```

Comment:
<p>retro woman artwork? I'm not sure what this is. Aren't we looking for a beaver? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>